test/front-matter-ignore-line.md

```diff
@@ -0,0 +1,9 @@
+---
+a: b
+c: d
+e: f
+g: h
+---
+
```

Comment:
Please add another kind of violation and suppression below, after a line with a valid reported violation via {MDxxx}, please.

Reply:
For example:

Front matter

Disabled violation

Violation

Disabled violation